packages/ui/src/ui/components/EditTextWithPreview/EditTextWithPreview.tsx

```diff
@@ -54,6 +57,8 @@ export function EditTextWithPreview({
     disabled,
     initialSplitSize,
 }: EditTextWithPreviewProps) {
+    const vimMode = useSelector(getSettingsMonacoVimMode);
```

Comment:
It is not allowed to use `redux`-related feature in `src/components` (like `useSelector`).
The component is used in several places of UI, but the option is available as `Settings/Queries/Vim mode`.

Reply:
As I understand this part is still actual:
`The component is used in several places of UI, but the option is available as Settings/Queries/Vim mode.`